backend/futuboard/views/swimlaneViews.py

```diff
@@ -32,6 +32,18 @@ def swimlanecolumns_on_column(request, column_id):
         return JsonResponse(serializer.data, safe=False)
 
 
+@api_view(["GET"])
+def get_actions_by_columnId(request, column_id):
+    if request.method == "GET":
+        try:
+            ticketIds_query_set = Ticket.objects.filter(columnid=column_id)
+            query_set = Action.objects.filter(ticketid__in=ticketIds_query_set)
+            serializer = ActionSerializer(query_set, many=True)
+            return JsonResponse(serializer.data, safe=False)
+        except:  # noqa: E722
+            raise Http404("Column actions not found")
```

Comment:
Tän pitäis olla sillä tavalla, että antaa 404 vaan sillä tietyllä errorilla, että esim. columnia ei löydy. Ei siis catchata kaikkia mahdollisia erroreita, ja antaa 404.